### DISCUSSION TITLE
No stream stats is displayed with greenlight

### DISCUSSION THREAD

**Discussion by csuned** [unknownskl/xbox-xcloud-client#806] [Q&A]
Dear Committee,

I was working on QoS measurement of XCloud Streaming. When try to show the debug statistics, it just gives me the 'disconnect' option. 

Could you please show me how to export the debug statistics? I was using the latest version 2.0.0-beta 7 on an Ubuntu machine.

I also tried 1.2.0 version and it simply stop on the loading process, so I did not get any chance to use the older version.

Many thanks,
Chuanhao

**Comment by carterlasalle** [comment]
same with me @unknownskl @csuned
